Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -65,22 +69,31 @@ There are usually two release engineers working at any given time. It goes witho
 **Phase 5: Closure**
 <br/>	*It starts after the app is accepted by Apple and final internal approval*
 
-1. Send the build to Testflight Beta (external testing). Select the `External Testers` group.
+1. Send the build to TestFlight Beta (external testing). Select the `External Testers` group.
 1. Press `Release this version` in App Store Connect
-1. Tag the release commit and create a GitHub release. Attach the binary as an artefact to the GitHub release (if you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
+1. Create tag(s) on the release commit and create GitHub releases for each app
+  - Make sure you create one tag and one GitHub release for each app released on the AppStore (eg. one for Babylon and one for Telus, in the most common case of joint releases of those two – even if they end up pointing to the same commit)
```

Comment:
I can see Telus being mentioned alongside Babylon everywhere, how does it work for other apps like NHS111?  It seems Telus should be mentioned alongside those apps. 

Since release engineer typically does not handle Telus releases, it could potentially cause confusion. 

Also, Telus does not release every time main Babylon app is released

Reply:
My original (erroneous) understanding was that Telus and Babylon were released in tandem, and I thus assumed that it was done during the same cycle and by the same release engineer, doing both at the same time, and from the same branch; hence the mention of a common `release/x.y.z` branch for both, and the joint mention. While for Bupa/NHS111/... I already noted it was totally out of the release cycle and separate cases.

After gathering some more info (especially from you 🙇) it appears my assumption was wrong and that Telus is in fact to be considered a separate release process by a separate release engineer, so no different from Bupa / NHS111 / ... (even if we likely release Telus more often than Bupa/NHS111).

I'll amend the document with this new information to propose instead that:

 - Since we don't need a special case with a common branch from which multiple apps (Babylon+Telus) would be published, but instead have separate release branches even for Telus (whose release branch is cut off of the Babylon's release branch once QA'd), I'll remove the case of the `release/x.y.z` branch name
 - **This mean we'll not use the `release/x.y.z` name anymore, and instead always use `release/{app}/{version}`** naming – even for Babylon and Telus

So a typical release cycle which happen to plan to both release Babylon and Telus for a version 1.2.3 would mean creating a `release/babylon/1.2.3` release branch first, wait for QA to sign it off, then create the `release/telus/1.2.3` branch from the `release/babylon/1.2.3` branch and make QA sign off the Telus-specific additions.